packages/ui-components/src/Button/Button.module.scss

```diff
@@ -0,0 +1,178 @@
+@import "../styles/tokens.scss";
+
+.crl-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-width: 1px;
+  border-radius: $crl-border-radius;
+  outline: none !important; // focus indicators added below
+  @include crl-transition(background-color, border-color, box-shadow);
+
+  // TODO(j-low): update with type tokens when available
+  font-size: 12px;
+  line-height: 14px;
+
+  &:hover {
+    cursor: pointer;
+
+    &:disabled {
+      cursor: default;
+
+      &:hover {
+        cursor: not-allowed;
+      }
+    }
+  }
+
+  &:focus {
+    &:active {
+      box-shadow: none;
+    }
+  }
+
+  :disabled {
+    &:hover {
+      cursor: not-allowed;
+    }
+  }
+
+  &.type-primary-success {
+    background-color: $crl-base-green;
+    color: $crl-base-white;
+    border-width: 1px;
+    border-style: solid;
+    border-color: $crl-base-green;
+
+    &:hover {
+      background-color: $crl-green-4;
+      color: $crl-base-white;
+    }
+
+    &:active {
+      background-color: $crl-green-4;
+      border-width: 1px;
+      border-color: mix(black, $crl-green-4, 10%);
+    }
+
+    &:focus {
+      background-color: $crl-base-green;
+      border-width: 1px;
+      border-color: mix(black, $crl-green-4, 20%);
+      @include crl-box-shadow(0, 0, 0, 3px, rgba($crl-base-green, 0.4));
+    }
+
+    &:disabled {
+      background-color: $crl-green-2;
+      border-color: $crl-green-2;
+    }
```

Comment:
I wanted to DRY this code out as there is a lot of near-repetition in the related classes below but it turns out there actually isn't consistent relationships between the BG/border/color changes across element states across button types that would lend itself to abstracting the login to a handy mixin.  Alas.